core/templates/dev/head/pages/learner_dashboard/LearnerDashboard.js

```diff
@@ -191,6 +227,100 @@ oppia.controller('LearnerDashboard', [
       return value;
     };
 
+    $scope.sortFeedbackFunction = function(entity) {
```

Comment:
I have no idea what `entity` means here, and I also don't know what "sortFeedbackFunction" is supposed to do from the name. Could you use names that are more illustrative of what these actually are?